### PR TITLE
Fix totp registration

### LINKED ISSUE #6961: MFA Management -> add new TOTP key gives blank screen.
### Describe the bug

Attempts to add TOTP 2FA fail.

### To Reproduce

1)  User (upper right corner) -> MFA Management -> Add New... -> TOTP Key
2)  Enter password
3)  Blank screen is then presented.
4)  The following appears in the error logs:

[Wed Nov 01 14:01:43.056570 2023] [php:error] [pid 852] [client 172.21.12.170:59463] PHP Fatal error:  Uncaught TypeError: RobThree\\Auth\\TwoFactorAuth::__construct(): Argument #4 ($algorithm) must be of type RobThree\\Auth\\Algorithm, string given, called in /var/www/localhost/htdocs/openemr/library/classes/Totp.class.php on line 63 and defined in /var/www/localhost/htdocs/openemr/vendor/robthree/twofactorauth/lib/TwoFactorAuth.php:30\nStack trace:\n#0 /var/www/localhost/htdocs/openemr/library/classes/Totp.class.php(63): RobThree\\Auth\\TwoFactorAuth->__construct()\n#1 /var/www/localhost/htdocs/openemr/interface/usergroup/mfa_totp.php(164): Totp->generateQrCode()\n#2 {main}\n  thrown in /var/www/localhost/htdocs/openemr/vendor/robthree/twofactorauth/lib/TwoFactorAuth.php on line 30, referer: http://docker.rtl.lan:8300/interface/usergroup/mfa_totp.php?action=reg1&error=auth

### Expected behavior
TOTP QR code presented.

### Client configuration

- Browser: doesn't matter -- chrome / firefox / vivaldi.
- OpenEMR version: 7.0.2-dev
- Operating system:  Docker

### PR DESCRIPTION
**Pull request by DrBassman**
Fixes #6961

#### Short description of what this resolves:
On the latest development version, attempts to add TOTP 2FA for the logged in user fail.

#### Changes proposed in this pull request:
Update library/classes/Totp.class.php to fix error messages.

### CODE REVIEW

File: library/classes/Totp.class.php [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 use RobThree\Auth\Providers\Qr\BaconQrCodeProvider;
 use RobThree\Auth\TwoFactorAuth;
+use RobThree\Auth\Algorithm;
```
Comment (stephenwaite) [inline library/classes/Totp.class.php:17]: nice fix @DrBassman ! 

1 minor thing that is mostly followed is the alpha ordering of the use statements

```
use RobThree\Auth\Algorithm;
use RobThree\Auth\Providers\Qr\BaconQrCodeProvider;
use RobThree\Auth\TwoFactorAuth;

Comment (DrBassman) [reply library/classes/Totp.class.php:17]: I'm still new to this & appreciate help "getting up to speed"... Do I update my changes so the use statements are in the preferred order?

Also, each time I do this should I delete my repository, and start with a newly forked version next time.  I'm a little confused how the message here says I have 6 commits to merge, when I only changed the one file?

Thank you for your help.

Comment (stephenwaite) [reply library/classes/Totp.class.php:17]: sure, update the use statements if you'd like in the next commit on this branch

no need to delete anything :scream: 

next time start a new branch after you update the `master` branch

anyhow, when we go to bring in your PR all of the commits will get squashed into 1 commit so it isn't really an issue anyways :)

**Review by stephenwaite — APPROVED** [review]

**Comment by stephenwaite** [comment]
jumping

**Comment by bradymiller** [comment]
2DV(5)

**Pull request merged**